src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -160,4 +170,36 @@ public void checkEntitiesWithMetadata() throws IOException {
 
       Assert.assertThat(actual, CoreMatchers.is(expected));
   }
+
+  @Test
+  public void sentimentIsNeutral() {
+    String actual = AnalysisHelper.getInterpretation(NEUTRAL_SCORE, NEUTRAL_MAGNITUDE);
+    String expected = NEUTRAL_INTERPRETATION;
+
+    Assert.assertEquals(expected, actual);
```

Comment:
In here you could also test a non neutral score with a neutral magnitude

Reply:
It should be impossible to have a non neutral score and neutral magnitude. Magnitude actual shows how much sentiment we have in the text somehow. Or at least the [documentation](https://cloud.google.com/natural-language/docs/basics#interpreting_sentiment_analysis_values) makes it seem so.